pkg/apis/build/v1beta1/buildrun_types.go

```diff
@@ -238,9 +238,15 @@ type BuildRunStatus struct {
 
 	// TaskRunName is the name of the TaskRun responsible for executing this BuildRun.
 	//
+	// Deprecated: Use ImageBuildrun instead to describe the taskrun.
 	// +optional
 	TaskRunName *string `json:"taskRunName,omitempty"`
 
+	// ImageBuildrun is the name and kind of the resource responsible for executing this BuildRun.
+	//
+	// +optional
+	ImageBuildrun *ImageBuildrun `json:"imageBuildrun,omitempty"`
```

Comment:
Introducing a different field generally makes sense. The field name however imo does not fit. If I were asking me the question "what is the ImageBuildrun of a BuildRun ?", I would not know what it is about.

A couple of ideas for alternatives:

- Implementation
- Executor
- Performer
- Driver

Please bring it up in a community meeting to brainstorm in a larger round.